fabric/src/main/java/me/shedaniel/architectury/registry/trade/fabric/TradeRegistryImpl.java

```diff
@@ -26,7 +26,7 @@
 import java.util.Collections;
 
 public class TradeRegistryImpl {
-    private static void registerVillagerTrade0(VillagerProfession profession, int level, VillagerTrades.ItemListing... trades) {
+    public static void registerVillagerTrade0(VillagerProfession profession, int level, VillagerTrades.ItemListing... trades) {
```

Comment:
Could you add @ApiStatus.Internal to both of these and the base method to let people know they *should not use this* (as it doesn't perform input validation)

Reply:
The common method is private and platform impls can be assumed to be internal (IMO), so I don't think it's needed necessarily.

Reply:
I also don't think it is needed. When I look at other platform impl, none of them is marked as internal